doc/source/developer/debug.md

```diff
@@ -41,17 +41,25 @@ The procedure from download to build is the following:
 ```console
 $ mkdir -p ~/work/
 $ cd ~/work/
-$ wget https://downloads.mariadb.org/rest-api/mariadb/11.4.3/mariadb-11.4.3.tar.gz
-$ tar xf mariadb-11.4.3.tar.gz
-$ rm -rf mariadb-11.4.3/storage/mroonga
+$ minor_version=$(curl -s https://downloads.mariadb.org/rest-api/mariadb/ \
```

Comment:
```suggestion
$ major_minor_version=$(curl -s https://downloads.mariadb.org/rest-api/mariadb/ \
```

Or `release_name`?

Reply:
fix: [f2aa453](https://github.com/mroonga/mroonga/pull/907/commits/f2aa453f4f4e8465e996813ef2bc393473639a58) Thanks. I used the `release_id` for consistency.